config-wdio/chrome-selenoid.conf.ts

```diff
@@ -0,0 +1,32 @@
+var defaults = require("./base.conf").config;
+var _ = require("lodash");
+import * as configVal from "config";
+const BASEURL = configVal.get("Environment.baseUrl");
+
+var overrides = {
+  baseUrl: BASEURL,
+  hostname: "localhost",
+  port: 4444,
+  path: "/wd/hub",
+  runner: "local",
+  logLevel: "info",
+  maxInstances: 1,
+
+  capabilities: [
+    {
+      browserName: "chrome",
+      browserVersion: "99.0",
```

Comment:
Is it possible to pass the dynamic version value instead of static browser version. Also, if possible can we keep this as latest?

Reply:
As of now it is not working with latest string, tried multiple options. For now, we are going ahead with static version. Will update in future if we find any better solution.